src/scvi/dataloaders/_data_splitting.py

```diff
@@ -83,6 +144,9 @@ class DataSplitter(pl.LightningDataModule):
     pin_memory
         Whether to copy tensors into device-pinned memory before returning them. Passed
         into :class:`~scvi.data.AnnDataLoader`.
+    external_indexing
+        A list of data split indexes in the order of training, validation, and test sets.
+        Validation and test set and not required and can be left empty.
```

Comment:
and not = are not.
Left empty = setting it to None or not using it?

Reply:
Both will work actually.